contrib/kind-common

```diff
@@ -129,7 +129,7 @@ install_metallb() {
   builddir=$(mktemp -d "${METALLB_DIR}/XXXXXX")
 
   pushd "${builddir}"
-  git clone https://github.com/qinqon/metallb.git
+  git clone https://github.com/metallb/metallb.git
```

Comment:
this was introduced in the previous commit ... can't this be squashed into it ? 

Having a build from your personal repo is weird.

Reply:
Totally, Fede is preparing a proper metallb release though so we don't have to pin to a commit id.